utility/src/main/java/org/oppia/android/util/parser/html/ImageTagHandler.kt

```diff
@@ -101,4 +101,11 @@ class ImageTagHandler(
       "Failed to parse $CUSTOM_IMG_CAPTION_ATTRIBUTE"
     )
   }
+
+  override fun getContentDescription(attributes: Attributes): String? {
+    val altValue = attributes.getJsonStringValue("alt-with-value")
+    return if (altValue != null) {
```

Comment:
Prefer using `isNullOrBlank()`, which is a more strict check and will prevent issues like https://github.com/oppia/lessons-team/issues/719 from re-curring.